roles/zuul/tasks/main.yml

```diff
@@ -164,6 +165,53 @@
     - Reload systemd units
     - Restart zuul
 
+- name: Create zuul status lib dir
+  file:
+    path: /opt/source/zuul/etc/status/public_html/lib
+    state: directory
+
+- name: Install jquery and jquery-visibility
+  get_url:
+    url: "{{ item }}"
+    dest: /opt/source/zuul/etc/status/public_html/lib
+  with_items:
+    - http://code.jquery.com/jquery.min.js
+    - https://raw.githubusercontent.com/mathiasbynens/jquery-visibility/master/jquery-visibility.js
+
+- name: Download jquery-graphite
+  get_url:
+    url: https://github.com/prestontimmons/graphitejs/archive/master.zip
+    dest: /tmp/jquery-graphite.zip
+  register: jquery_graphite
+
+- name: Unpack jquery-graphite
+  unarchive:
+    remote_src: yes
+    src: /tmp/jquery-graphite.zip
+    dest: /tmp/
+  when: jquery_graphite|changed
```

Comment:
jquery graphite is a one file app, can we do the same thing here as visibility and just download https://raw.githubusercontent.com/prestontimmons/graphitejs/master/jquery.graphite.js